patroni/dcs/__init__.py

```diff
@@ -1080,42 +1085,50 @@ def _get_members_slots(self, my_name: str, role: str) -> Dict[str, Dict[str, str
                                    for k, v in slot_conflicts.items() if len(v) > 1))
         return slots
 
-    def has_permanent_logical_slots(self, my_name: str, nofailover: bool, major_version: int = 110000) -> bool:
+    def has_permanent_slots(self, my_name: str, nofailover: bool = False) -> bool:
```

Comment:
Not a critique but rather a query so I know how better to review in the future.

Are we concerned at all about anyone who might be using patroni as a standard library which they might implement their own extensions from? I am thinking if a customer has say an internal key/value store and they want to write their own DCS module.

Reply:
While in theory anyone could implement their own, on practice I don't expect in ever happen, so we support only DCS that are included in `patroni/dcs`.

Reply:
Ok cool, thanks for confirming.